docs/config.md

```diff
@@ -218,6 +218,7 @@ topic          |     Y     |  string     | N/A     |
 codec          |     Y     |  string     | N/A     | which compression to use. possible values: none, gzip, snappy
 partitionBy    |     Y     |  string     | N/A     | which fields to shard by. possible values are: byOrg, bySeries, bySeriesWithTags
 schemasFile    |     Y     |  string     | N/A     |
+kafkaVersion   |     N     |  string     | ""      | Kafka version in semver format. All brokers must be this version or newer. (fallback to oldest stable version 1.0.0)
```

Comment:
go.mod pins sarama to v1.23.0 
That version's ParseKafkaVersion function returns V0_8_2_0 if no version could be parsed